jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -180,8 +180,11 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_FALSE, "false")
 #if !defined (CONFIG_DISABLE_MATH_BUILTIN)
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_FLOOR, "floor")
 #endif
-#if !defined (CONFIG_DISABLE_REGEXP_BUILTIN)
+#if !defined (CONFIG_DISABLE_REGEXP_BUILTIN) \
+|| !defined (CONFIG_DISABLE_STRING_BUILTIN)
```

Comment:
This file is automatically generated by the `gen-magic-strings.py` script. I can modify this but it will also change the other occurrences. I will create a follow-up patch for that. What you think?